BookingsApi/BookingsApi/Controllers/JusticeUserController.cs

```diff
@@ -54,8 +54,9 @@ public async Task<IActionResult> AddJusticeUser(AddJusticeUserRequest request)
                 ModelState.AddFluentValidationErrors(validation.Errors);
                 return ValidationProblem(ModelState);
             }       
+            int[] userRoleIds = request.Roles.Select(x => (int) x).ToArray();
```

Comment:
again, here, a var instead of the type would be fine because the right hand side of the = statement is apparent enough
```suggestion
            var userRoleIds = request.Roles.Select(x => (int) x).ToArray();
```

Reply:
same as above, I would recommend changing it

Reply:
changed